modules.json

```diff
@@ -36,8 +36,8 @@
                         "installed_by": ["modules"]
                     },
                     "deseq2/differential": {
-                        "branch": "master",
-                        "git_sha": "25047aa940979f64b31d19b94c483a9254263892",
+                        "branch": "deseq2_transcript_lengths",
```

Comment:
Why not from master?

Reply:
It will be, just wanted to test while we solved the issues in https://github.com/nf-core/modules/pull/4376